src/main/java/org/alliancegenome/curation_api/services/InformationContentEntityService.java

```diff
@@ -28,4 +28,8 @@ public InformationContentEntity retrieveFromDbOrLiteratureService(String curieOr
 
 		return ice;
 	}
+
+	public InformationContentEntity retrieveFromDbOrLiteratureServicea(String pmid) {
```

Comment:
Can this be renamed to something more descriptive to indicate why it is different from `retrieveFromDbOrLiteratureService()`

Reply:
Wasn't used any longer. Removed it.